clj/src/cljd/flutter/alpha.cljd

```diff
@@ -29,14 +29,14 @@
     (if watch
       (cond->>
         `(widgets/StatefulBuilder.
-           :key ~(cond-> key (contains? opts :key) (list `foundation/ValueKey.))
+           :key ~(when key (list `foundation/ValueKey. key))
```

Comment:
or may be there is a reason to use cond? It could be rewritten with `cond->>`

Reply:
It should have been `cond->>` indeed. The intent was to have a behavior different between no `:key` and `:key` set to `nil`.